sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -27,6 +33,22 @@ applyJavaNature(
   ],
 )
 
+// For resolving Cloud Bigtable dependencies from custom registry that includes Change Stream API.
+repositories {
+  maven {
+    url "artifactregistry://us-central1-maven.pkg.dev/cloud-bigtable-ecosystem/bigtable-change-streams-preview"
+  }
+}
+configurations.all {
+  resolutionStrategy {
+    force group: 'com.google.cloud', name: 'google-cloud-bigtable', version: '2.11.2-change-streams-preview1-SNAPSHOT'
```

Comment:
since these are all SNAPSHOT versions, they should not appear in beam release versions. What is the time line that it will get released? If is within a month or so, could you please create a GitHub issue and set milestone to v2.46.0 as a release blocker?

Reply:
This is merging into a feature branch. None of this will be included in when merging into master branch. This is a temporary work around to get code logic reviewed/compile/pass tests while we wait for Cloud Bigtable's java client to catch up to include the new API.

We will remove all this before creating the PR to merge into master.

Reply:
ah I see, the base branch is not master, sorry for confusion

Reply:
Also created https://github.com/apache/beam/issues/25412 to track this feature work.